### DISCUSSION TITLE
Conflict (line break)

### DISCUSSION THREAD

**Discussion by aguiarcc** [twbs/bootstrap#34965] [General]
I'm using the flexbox classes inside a column ( d-flex align-items-center justify-content-left ) to control the placement of the texts ( h1 and p ), but using flexbox to conflict with the content line break. paragraph tag. How do I get the content of the paragraph below the content of the h1 tag? Below is the code:

```
<div class="col d-flex align-items-center justify-content-left">
 <h1>App Name</h1>
 <p>Content em desenvolvimento</p>
</div>
```

**Comment by ghost** [comment]
I couldn't understood your problem but yes, you can try the following
1) Setting col width(col-1,col-2,col-3.....)
2) Try Justify-content-start instead of justify-content-left
3) Try putting alignment classes on h1 and p tag.

I hope it helps.

**Reply by aguiarcc** [reply]
Did not work

**Reply by ghost** [reply]
> Did not work

Ok. I read it in documentation that it will work if
flex-direction:column; as css entry
